ush/forecast_postdet.sh

```diff
@@ -675,13 +675,19 @@ GOCART_rc() {
   # set input directory containing GOCART input data and configuration files
   # this variable is platform-dependent and should be set via a YAML file
 
+  local vdate
   # link directory containing GOCART input dataset, if provided
   if [[ -n "${AERO_INPUTS_DIR}" ]]; then
     ${NLN} "${AERO_INPUTS_DIR}" "${DATA}/ExtData"
     status=$?
     [[ ${status} -ne 0 ]] && exit "${status}"
   fi
 
+  # Link blending emissions if AERO_EMIS_FIRE == blending
+  if [[ "${AERO_EMIS_FIRE}" == "blending" && "${RUN}" == "gefs" ]]; then 
```

Comment:
I'm not sure... Rahul asked to add this but

Reply:
Since this was only GEFS related, I suggested the `RUN` condition check.  Pretty sure, SFS will need that as well in the future.
I can go either way.